src/book/03-guides/04-general/03-glossary.mdx

```diff
@@ -5,3 +5,181 @@ title: Glossary
+
+[**NUbook**](https://nubook.nubots.net): The NUbots documentation website. Source is hosted on [GitHub](https://github.com/NUbots/NUbook).
+
+[**NUClear**](https://github.com/Fastcode/NUClear): The software framework which the NUbots codebase is built on. Put simply, it works by triggering callbacks when events occur. Documentation for NUClear is [here](https://nuclear.readthedocs.io/en/latest/index.html).
+
+[**NUClear Binary Stream (NBS)**](https://nubook.nubots.net/system/foundations/build-system#nbs): NUClear logging file format, which is basically concatenated protobuf messages with headers.
+
+[**NUClear Powerplant**](https://nuclear.readthedocs.io/en/latest/components.html#powerplant): The central message system which connects the reactors.
+
+[**NUClear Reactor**](https://nuclear.readthedocs.io/en/latest/components.html#reactors): A NUClear module. Modules in the NUbots codebase are reactors.
+
+[**NUClear Reaction**](https://nuclear.readthedocs.io/en/latest/components.html#reactions): The callbacks which need to run when an event or conditions indicate as such. These manifest as DSL [on statements](https://nuclear.readthedocs.io/en/latest/dsl.html#on-statements).
+
+[**NUClear Roles**](https://github.com/Fastcode/NUClearRoles): The build and messaging system used to manage the construction and messaging of NUClear for the NUbots codebase.
+
+**NUFSR**: A system designed to detect if a NUgus' foot is on the ground, by use of force sensitive resistors (FSRs).
+
+[**NUgan**](https://github.com/NUbots/NUgan): A custom [cycle-gan](https://junyanz.github.io/CycleGAN/) implementation to generate more realistic images for training the vision networks.
+
+[**NUgus**](https://nubook.nubots.net/system/hardware/overview#igus-modifications): NUbots robots as currently used in the RoboCup humanoid league. They are modified [igus Humanoid Open Platform](https://arxiv.org/abs/1809.11110) robots, which we 3D-printed.
+
+[**NUpbr**](https://github.com/NUbots/NUpbr): The software NUbots uses to generate Physically Based Rendered football fields, used to train vision networks.
+
+[**NUsight**](https://github.com/NUbots/NUsight2): NUbots' visualisation software, ran through a browser on an external computer.
+
+**NUsense**: A secret project for sensor IO.
+
+**Old Walk Engine**: A previously used walk engine, originally optimised using genetic algorithms.
+
+**Quintic Walk**: Open loop walk engine which uses quintic splines to create trajectories. Created by Bit-Bots, based off code from team Rhoban, then ported to run with NUClear.
+
+**ScriptTuner**: A NUbots program for manually creating motion targets for movement scripts.
+
+**ScriptRunner**: The program which runs movement scripts on the robot, such as `Zombie.yaml` which is used to test servo installations.
+
+**Static Motion**: We tend to refer to motion which is fully controlled by scripts which do not change at run-time as _Static Motion_.
+
+**Subsumption**: TODO!
```

Comment:
```suggestion
**Subsumption**: The priority system that determines if a neutron gets access to the servos it wants to control.
```
Might need rewording. But something like this.

Reply:
We should probably briefly describe what a neutron is here in parenthesis, or add a new term in the glossary and link to that, or link to a definition somewhere in the NUClear docs.